### PR TITLE
Change ModListItem drag button to div

### PR DESCRIPTION
**Pull request by estebanfer**
This would fix the issue of the mods not being dragged when pressing on the padding of the drag handle

### CODE REVIEW

File: src/tauri/src/lib/pages/playlunky/ModListItemDragHandle.svelte [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+</script>
+
+<div
+  {...$$props}
+  on:mousedown
+  on:mouseup
+  on:touchstart
+  on:touchend
+  class="{buttonStyles.base}
+    {buttonStyles.size['default']}
```
Comment (mauvealerts) [inline src/tauri/src/lib/pages/playlunky/ModListItemDragHandle.svelte:14]: I mildly prefer dot-notation (`buttonStyles.size.default`) over bracket notation. (Same applies to `transparent` below. WDYT?

**Review by mauvealerts — COMMENTED** [review]
Thanks! This looks good with one tiny comment

File: src/tauri/src/lib/pages/playlunky/ModListItemDragHandle.svelte (hunk 1 above)
Comment (estebanfer) [reply src/tauri/src/lib/pages/playlunky/ModListItemDragHandle.svelte:14]: ah true it should be better like how you said, I didn't really think about that, I just copied that part of code from the button component 😅

Comment (estebanfer) [reply src/tauri/src/lib/pages/playlunky/ModListItemDragHandle.svelte]: Should I squash the commit? idk how it works after already pushing the changes

**Comment by mauvealerts** [comment]
I can squash while merging 😃 Thanks for the fix!

**Pull request merged**